bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -0,0 +1,887 @@
+            }
+        })
+        //if meeting is a onetime(2) meeting
+        if (meeting.type == 2) {
+            dataStore.save('meeting_details_one_time', meetingDetails)
+            dataStore.save('meeting_details_one_t_1', dialIn);
+            dataStore.save('meeting_details_one_t_2', dialInNumbers);
+        }
+        //if meeting type is recurring(8)
+        else if (meeting.type == 8) { 
+            let occurrenceData = meeting.occurrences.map(occurrence => {
+                return {
+                    "duration": occurrence.duration,
+                    "occurrence_id": Number(occurrence.occurrence_id),
+                    "meetingId": meeting.id,
+                    "start_time": new Date(occurrence.start_time),
+                    "parent_id": meeting.id,
+                    "visibility": true
+                }
+            })
+            //Storing the data to the cache DB            
+            dataStore.save('meeting_details_recurring', meetingDetails);
+            dataStore.save('meeting_details_recur_1', occurrenceData);
+            dataStore.save('meeting_details_recur_2', dialIn);
+            dataStore.save('meeting_details_recur_3', dialInNumbers);
+        }
+        else{
+            console.log("Found a different meeting other than onetime and reucrring meeting id: "+meeting.id+" type: "+meeting.type)
+        }
+    }
+    return meetingsMap;
+}
+
+// Loading the Meeting's Invitation
+async function syncMeetingInvitations(dataStore, client,meetingdetail) {
+    //traversing through the meetingMap to get meeting id as key and email as value
+    for (const [meeting, user] of meetingdetail.entries()) {
+        let invitationreq = await client.fetch(`meetings/${meeting}/invitation`);
+        if (!(await invitationreq.ok)) {
+            throw new Error(JSON.stringify(await invitationreq.json()));
+        }
+        let invitationres = await invitationreq.json();
+        dataStore.save("meeting_invitations", {
+            "id": meeting,
+            "invitation": invitationres.invitation,
+            "users_email": user.email
+        })
+    }
+}
+
+async function syncRecordings(dataStore, client,userids) {
+    let i = 0;
+    //defining the 7 days window to fetch the recordings data for an user
+    let to = moment().utc().format();
+    let from = moment().subtract(7, 'days').utc().format();
+    let next_page_token = "";
+    const page_size = 100;
+
+    do {
+        let response = await client.fetch(`users/${userids[i].userid}/recordings?from=${from}&to=${to}&next_page_token=${next_page_token}&page_size=${page_size}`);
+        if (!(await response.ok)) {
+            throw new Error(JSON.stringify(await response.json()));
+        }
+        let responseData = await response.json();
+        next_page_token = responseData['next_page_token'] ?? "";
+        let recordings = responseData.meetings;
+        let recordDatacounter = 0;
+        let data = recordings.map(meeting => {

```

Comment:
```suggestion
        let recordDatacounter = 0;
        const data = responseData.meetings.map(meeting => {
```

Reply:
In these few comments, I wanted to encourage you to use `const` more.
Of course that `let` works too, but when you read code with `const` then you know that this variable (reference) won't be changed later.